pkg/util/azureclient/environments.go

```diff
@@ -0,0 +1,50 @@
+package azureclient
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"fmt"
+	"strings"
+)
+
+// AROEnvironment contains additional, cloud-specific information needed by ARO.
+// Based on Environment from github.com/Azure/go-autorest/autorest/azure
+type AROEnvironment struct {
+	ActualCloudName          string
+	GenevaMonitoringEndpoint string
+	AppSuffix                string
+	CISubscriptionID         string
+	CIResourceGroup          string
+}
+
+var (
+	// PublicCloud contains additional ARO information for the public Azure cloud environment.
+	PublicCloud = AROEnvironment{
+		ActualCloudName:          "AzureCloud",
+		GenevaMonitoringEndpoint: "https://gcs.prod.monitoring.core.windows.net/",
+		AppSuffix:                "aro.azure.com",
+		CISubscriptionID:         "46626fc5-476d-41ad-8c76-2ec49c6994eb",
```

Comment:
Not the fan of these being here, but if this is a blocker we could merge but have follow-up issue to parameterize those. Considering both `CISubscriptionID` and `CIResourceGroup` on ly CI artifacts, they should not be in production code.

Reply:
Are you saying to move just `CISubscriptionID` and `CIResourceGroup` to env vars set in RP config? I agree, that would be cleaner.

Reply:
I'll leave the CI related work for a future PR that uses env vars